pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -199,6 +199,7 @@ func (h *heartbeatSender) startHeartbeatLoop(ctx context.Context) {
 				}
 				sent, streamStatus, err := h.maybeHeartbeat(ctx, h.frontier)
 				// TODO(casper): add unit tests to test different kinds of client errors.
+				// Source cluster may be unavailable
```

Comment:
Is this comment useful?  If so, super minor nit is that we try to have punctuated, complete sentences in our comments:

```suggestion
				// The source cluster may be unavailable.
```